region/crateria/east.json

```diff
@@ -836,6 +864,32 @@
               ],
               "openEnd": 1
             }
+          ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "West Ocean: G-Mode Setup Lure Zeb from Below (to Bowling Alley Top)",
+                  "notable": true,
+                  "requires": [
+                    "canTrickyUseFrozenEnemies",
+                    {"or": [
+                      "h_canBombThings",
+                      "h_canUseSpringBall"
+                    ]},
+                    "canCameraManip"
+                  ],
+                  "note": [
+                    "Go down to the Zeb Farm which normally leads to the Bowling Alley Path.",
+                    "While standing on the right of the Zeb farm, freeze the Zeb while it is still moving up and facing to the right.",
+                    "Quickly go through the morph tunnel with Bombs or Spring Ball, and get onto the small floating platform just above the Zeb farm.",
+                    "After the Zeb is visible, fall off the platform to make the Zeb move horizontally at the right height to hit Samus in the doorway to the right.",
+                    "Note that if the Zeb is off screen horizontally it will despawn, but if it's off screen vertically it will not."
```

Comment:
```suggestion
                    "Note that if the Zeb is unfrozen and off screen horizontally it will despawn, but if it's off screen vertically it will not."
```